action.yaml

```diff
@@ -14,21 +14,80 @@ inputs:
   description:
     description: "Additional information for the commit"
     required: false
-    default: "${{ github.event.head_commit.message }}"
+    default: "Artefacts CI check"
+  artefacts-url-api:
+    description: "Artefacts API URL"
+    required: false
+    default: "https://app.artefacts.com/api"
   working-dir:
-    description: "Working dir containing "
+    description: "Working dir containing"
     required: false
     default: "."
```

Comment:
can we add a `wait_for_results` input parameters, that disable the wait for customer who would like the action to return immediately? with a default value of  `False`